test/third_party_tests/test_native_executor.py

```diff
@@ -83,6 +83,27 @@ def _create_evadb_table_using_select_query(self):
             "DROP TABLE IF EXISTS eva_table;",
         )
 
+    def _create_native_table_using_select_query(self):
+        execute_query_fetch_all(
+            self.evadb,
+            """CREATE TABLE test_data_source.derived_table AS SELECT name, age FROM test_data_source.test_table;""",
+        )
+        res_batch = execute_query_fetch_all(
+            self.evadb,
+            "SELECT * FROM test_data_source.derived_table",
+        )
+        self.assertEqual(len(res_batch), 2)
+        self.assertEqual(res_batch.frames["derived_table.name"][0], "aa")
+        self.assertEqual(res_batch.frames["derived_table.age"][0], 1)
+        self.assertEqual(res_batch.frames["derived_table.name"][1], "bb")
+        self.assertEqual(res_batch.frames["derived_table.age"][1], 2)
+        execute_query_fetch_all(
+            self.evadb,
+            """USE test_data_source {
+                DROP TABLE IF EXISTS derived_table
+            }""",
+        )
```

Comment:
Is the clean up code? If so, shall we put it in the cls teardown?

Reply:
I agree. The test class is getting complicated and needs to be simplified.